gutenberg.php

```diff
@@ -46,6 +46,21 @@ function the_gutenberg_project() {
 		<div id="metaboxes" class="hidden">
 			<?php the_block_editor_meta_boxes(); ?>
 		</div>
+		<?php
+		/**
+		 * Start: Include for phase 2
```

Comment:
We must find another way to extend this. It's slated for removal in #13569.

Reply:
Does it need to occur where it is in the markup here? Or could it be done as a hook handler to [`in_admin_footer`](https://developer.wordpress.org/reference/hooks/in_admin_footer/) ?